.github/files/generate-ci-matrix.php

```diff
@@ -89,6 +89,16 @@
 	'with-woocommerce' => true,
 );
 
+// Add WooCommerce tests.
+$matrix[] = array(
+	'name'         => 'PHP tests: PHP 7.4 WP latest with WPCOMSH',
+	'script'       => 'test-php',
+	'php'          => '8.1',
```

Comment:
I wonder if we can avoid hard-coding 8.1 here, to avoid having to remember to update this in the future when Atomic's minimum PHP version changes.

Might it make sense to read wpcomsh's composer.json for `.require.php`, which I think will probably be ">=8.1", and strip out the ">=" to determine the version to use?